.changes/next-release/feature-DynamoDBEnhancedClient-0c57ace.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "DynamoDB Enhanced Client", 
+    "contributor": "shetsa-amzn", 
+    "type": "feature", 
+    "description": "Adding a member to the UpdateItemEnhancedRequest.Builder named returnItems, that accepts values from the com.amazonaws.services.dynamodbv2.model.ReturnValue enum, and defaults to the current behaviour ALL_NEW, and pass it to the request instead of the constant ALL_NEW."
```

Comment:
Can we simplify description to something like:
Adds support for specifying ReturnValue in UpdateItemEnhancedRequest

And update the PR description as well?
com.amazonaws.services.dynamodbv2.model is v1 actually